rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -524,6 +523,28 @@
         }
       }
     },
+    "client_id": {
+      "title": "Client ID",
+      "description": "The client reference ID at the vendor.",
+      "type": "string",
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "order_reference": {
+      "title": "Order reference",
```

Comment:
Order number?

Reply:
Same as previous. Seems more generic to use `_reference`. @PascalRepond @ManaDeweerdt ?